{{cookiecutter.module_name}}/LICENSE

```diff
@@ -0,0 +1,201 @@
+Apache License
```

Comment:
The formatting seems to have gotten collapsed here, this is usually "centered". Was that intentional?

Reply:
Not intentional. Fixed